VTEX - Catalog API.json

```diff
@@ -2906,6 +2906,12 @@
                                         "title": "KitItensSellApart",
                                         "description": "Defines if Kit components can be sold apart",
                                         "default": false
+                                    },
+                                    "ActivateIfPossible": {
+                                        "type": "boolean",
+                                        "title": "ActivateIfPossible",
+                                        "description": "When set as `true`, this attribute will automatically update the SKU as active once associated with an image or an active component",
```

Comment:
```suggestion
                                        "description": "When set to `true`, this attribute will automatically update the SKU as active once associated with an image or an active component",
```

Reply:
https://english.stackexchange.com/questions/272165/set-as-default-set-to-default#:~:text=Here's%20the%20math%20way.,%2C%20lol)%20will%20be%20restored.